docs/en/observability/synthetics-migrate-integration.asciidoc

```diff
@@ -0,0 +1,153 @@
+* Elastic's global managed infrastructure
+
+| *Where you configure monitors*
+a| You could configure monitors using:
+
+* A user interface in {kib} (all monitors)
+* Code in an external, version-controlled project (browser monitors _only_)
+a| You can configure monitors using:
+
+* A user interface in {kib} (all lightweight monitors, browser monitors via inline script only)
+* Code in an external, version-controlled project (all monitors)
+
+| *How to use projects*
+(<<synthetics-migrate-integration-projects,read{nbsp}more>>)
+a| . Created a project that uses `@elastic/synthetics`.
+   . Wrote journeys in JavaScript or TypeScript files.
+   . Zipped the entire project.
+   . Created monitor using the Integrations UI.
+   . Configured the monitor in the UI including
+     a ZIP URL that pointed to the location of the project.
+a| . Create a project that uses `@elastic/synthetics`.
+   . Configure lightweight monitors in YAML files.
+   . Write journeys in JavaScript or TypeScript files and configure
+     individual monitors in your journey code using `monitor.use` or
+     configure all monitors using the `synthetics.config.ts` file.
+   . Use the `elastic/synthetics push` command to create monitors.
+
+Find more details in <<synthetics-get-started-project>>.
+
+| *How to use the UI*
+(<<synthetics-migrate-integration-ui,read{nbsp}more>>)
+a| . Went to the *Integrations* page in {kib}.
+   . Searched for and added the *Elastic Synthetics* integration.
+   . Configured the monitor.
+   . Created the monitor.
+a| . Go to the *{uptime-app}* in {kib}.
+   . Go to *Monitor Management*.
+   . Click *Add monitor*.
+   . Configure the monitor.
+   . Create the monitor.
+
+Find more details in <<synthetics-get-started-ui>>.
+|===
+
+[discrete]
+[[synthetics-migrate-integration-location]]
+= Where monitors run
+
+If you want to continue hosting on your infrastructure, you will need to create a
+private location before creating monitors.
+
+// Is this true?
+If you have already have an {agent} running using `elastic-agent-complete`,
+you can <<synthetics-private-location-add,add it as a new private location>>
+in the {uptime-app}.
+
+To create a new private location from scratch, follow all instructions in
+<<synthetics-private-location>>.
+
+Alternatively, you can start hosting on Elastic's global managed infrastructure.
+With Elastic's global managed testing infrastructure, you can create and run monitors in multiple
+locations without having to manage your own infrastructure.
+Elastic takes care of software updates and capacity planning for you.
+
+[discrete]
+[[synthetics-migrate-integration-projects]]
+= How to use projects
+
+// Is this true?
+If you already have an external project you were adding via a ZIP URL
+you can use the same project, but you will have to make some changes:
+
+. Update to the latest version of `@elastic/synthetics`.
+. Configure monitors with code instead of in the UI:
```

Comment:
I'm not sure I understand this.  As this is in the section about ZIP URLs, won't that mean they will have already defined them in code, as opposed to the UI?

Reply:
Yes I see that this is confusing. With the integration you _had_ to configure all the non-journey details (like the name of the monitor, the frequency at which it runs, throttling options, screenshots, etc) in the UI right? And with project monitors now you configure all those details in the code itself instead of in Kibana. If that's correct, do you have any thoughts on how to make that clearer?

Reply:
Ah yes, of course, thanks for the clarification.

I wonder if it would help to give some context here, such as `Configure monitors (such as the frequency) with code instead of in the UI` (or is this too wordy?)

To @andrewvc's [point](https://github.com/elastic/observability-docs/pull/2445#pullrequestreview-1205581794), it's also worth adding a bullet between the current (1) and (2) that describes the need to `npm @elastic/synthetics init...` the folder too.